src/domUpdates.js

```diff
@@ -15,7 +15,7 @@ const domUpdates = {
   },
 
   appendAnswers(game) {
-    console.log(game.round.turn.answers[0].respondents)
+    // console.log(game.round.turn.answers[0].respondents)
```

Comment:
Get in the habit of removing console.logs rather than commenting them out, so you don't forget to delete them down the line